src/index.coffee

```diff
@@ -3,8 +3,8 @@ Path  = require('path')
 Hubot = require('hubot')
 
 class MockRobot extends Hubot.Robot
-  constructor: ->
-    super null, null, false, 'hubot'
+  constructor: (httpd=false) ->
```

Comment:
httpd is enabled by [default](https://github.com/github/hubot/blob/51cf314175588018b31768c3d54e3cfee07f5151/bin/hubot#L27), so this should be true and we don't have to do `httpd: true` on all test.